examples/soundspeed/atlas/ex_atlases_auto_query.py

```diff
@@ -10,26 +10,76 @@
 from hyo2.soundspeedmanager.qt_callbacks import QtCallbacks
 
 
+class ModelOptions(IntEnum):
```

Comment:
Why are you duplicating this enumeration? You should just import and use the one in `regofs.py`.